Nyron.md

```diff
@@ -0,0 +1,171 @@
+
+The choices are `border`, `outline`, `padding`, and `margin`. You will leave two spaces blank.
+
+Your Answer:
+
+```text
+<<<<<<< HEAD
+__margin_: Defines the distance between an element's border and adjacent elements' borders.
+_outline__: Inserts a "wall" around an element.
+__padding_: Defines the distance between an element's content and its border.
+___: Defines the width of an element.
+___: Overlays a "wall" on top of an element.
+__border_: Defines the distance between the center of an element and the center of the adjacent element.
+=======
+___: Defines the distance between an element's border and adjacent elements' borders.
+___: Inserts a "wall" around an element.
+___: Defines the distance between an element's content and its border.
+___: Defines the width of an element.
+___: Overlays a "wall" on top of an element.
+___: Defines the distance between the center of an element and the center of the adjacent element.
+>>>>>>> 51399ad975c06be17dd5fc0bb04cdecf56bb5443
+```
+
+### Question #5
+
+What does the following selector do?  `ul.dropdown > li`?
+
+Select 1:
+```
+<<<<<<< HEAD
+[x] Selects all li's which are directly inside a ul of class dropdown (children)
+=======
+[] Selects all li's which are directly inside a ul of class dropdown (children)
+>>>>>>> 51399ad975c06be17dd5fc0bb04cdecf56bb5443
+[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[] Selects all ul's of class dropdown, as well as the children elements that are li's
+[] Selects all ul's of class dropdown, only if their children are exclusively li's
+```
+
+### Question #6
+
+Identify the three places CSS can go, and rank them in terms of specificity:
+
+```text
+<<<<<<< HEAD
+1. classes
+2. style
+3. element
+=======
+1.
+2.
+3.
+>>>>>>> 51399ad975c06be17dd5fc0bb04cdecf56bb5443
+```
+
+## Git
+
+### Question #7
+
+Which of the following represents a correct workflow for submitting a PR on a non-master branch?
+(ignore the lack of commit messages)
+
+Select 1:
+```
+<<<<<<< HEAD
+[x] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+=======
+[] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+>>>>>>> 51399ad975c06be17dd5fc0bb04cdecf56bb5443
+[] fork on github; git clone <ga_dc_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git push; create pull request
+[] git clone <ga_dc_url>; git branch <charlie_solution>; git add <files>; git commit; git push; create pull request
+[] fork on github; git clone <fork_url>; git checkout -b <charlie_solution>; git add <files>; git commit; git pull; create pull request
+```
+
+### Question 8
+
+What is the difference between a fork and a clone?
+
+<<<<<<< HEAD
+```A fork is copying the remote repository.
+  A clone is copying the repository to your local
+=======
```

Comment:
:+1:  A clone is a version of someone else's repository downloaded to your computer. A fork is a copy of someone else's repository, not necessarily downloaded to your computer. You can clone a fork; you can't fork a clone.